### DISCUSSION TITLE
Save name from excel cell?

### DISCUSSION THREAD

**Discussion by ghost** [clawsoftware/clawPDF#77] [General]
I love ClawPDF, i got it initially to allow me to save excel pages as pdf's with no metadata, something that other pdf printers were saving that i didn't want showing.

I'm wondering if it is possible to automatically set the file save name from the text in a cell in excel? 

i have an excel project where i change the data depending on a drop down selection box, and the numbers in that box is what i currently use for the save name when i save it as pdf, which is it's final output
i.e. 12345.pdf, 23456.pdf etc. 

I have to change the data in excel 24 times and save as pdf each time, then move onto the next worksheet and repeat etc, so it's very slow having to manually enter the save as name each time, not to mention i make mistakes quite often by mixing the number up.

I am useless at coding / programming, so i can't figure out if this is possible with ClawPDF,

**Comment by clawsoftware** [comment]
Hi @gazz29,

Thanks for your support. Unfortunately it doesn't work natively without a script. I have created an example script, which should meet your requirements: https://github.com/clawsoftware/clawPDF/blob/master/docs/com_examples/Powershell/Excel2PDF.ps1

You have to fill the option `$rangeSheet` according to your requirements. The script assumes that the row/column is the same on each worksheet.

How to run ps1: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_run_with_powershell

Kind regards,
clawsoftware
